orttraining/orttraining/python/checkpointing_utils.py

```diff
@@ -69,7 +69,7 @@ def _split_name(self, name):
         elif name_split[0].endswith("_fp16"):
             mp_suffix = "_fp16"
         param_name = name_split[0]
-        if optimizer_key != "":  # noqa: PLC1901
```

Comment:
Just for my learning: why we don't need this anymore without any update? Were they (PLC1901 errors) false positive or they are not required for the latest ruff anymore?